services/smtp/config.go

```diff
@@ -57,6 +59,8 @@ func (c Config) Validate() error {
 		return fmt.Errorf("invalid from email address: %q", c.From)
 	}
 	for _, t := range c.To {
+		// this is technically not correct, but I have only seen one email address without an `@` ever, in the modern day
+		// this is a safe, bet... no one uses bang paths anymore (also many libraries and services don't support them)
```

Comment:
How about using [mail.ParseAddress](https://pkg.go.dev/net/mail#ParseAddress) for more robust validation?